hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -125,10 +129,30 @@ instance FromCBOR Tx where
         (pure . fromLedgerTx)
 
 instance ToJSON Tx where
-  toJSON = toJSON . toLedgerTx
+  toJSON tx =
+    object
+      [ "cborHex" .= (Aeson.String $ decodeUtf8 $ Base16.encode $ serialiseToCBOR tx)
+      , "txId" .= (txId tx)
```

Comment:
Must: ensure that JSON transactions sent as response from `/commit` are compatible with `cardano-cli`

I do believe that this would be rejected due to a missing `type` (at least). I also could not find a test which would claim otherwise.